lib/shoryuken/message.rb

```diff
@@ -1,5 +1,16 @@
 module Shoryuken
   class Message
+    extend Forwardable
+
+    def_delegators(:data,
+                   :message_id,
+                   :receipt_handle,
+                   :md5_of_body,
+                   :body,
+                   :attributes,
+                   :md5_of_message_attributes,
+                   :message_attributes)
+
```

Comment:
A minor thing but skipping the parentheses in this case would match the macro-style methods better and also sorting the lines ascending would make it easier to read, e.g.:

```ruby
def_delegators :attributes,
               :body,
               :data,
               :md5_of_body,
               :md5_of_message_attributes,
               :message_attributes,
               :message_id,
               :receipt_handle
```